README.md

```diff
@@ -2,44 +2,64 @@
 
 [![Daily compatibility check against latest polkadot](https://github.com/paritytech/staking-miner-v2/actions/workflows/nightly.yml/badge.svg)](https://github.com/paritytech/staking-miner-v2/actions/workflows/nightly.yml)
 
-WARNING this library is under active development DO NOT USE IN PRODUCTION.
+This is a re-write of the [polkadot staking miner](https://github.com/paritytech/polkadot/tree/master/utils/staking-miner) using [subxt](https://github.com/paritytech/subxt) to avoid hard dependency to each runtime version.
 
-The library is a re-write of [polkadot staking miner](https://github.com/paritytech/polkadot/tree/master/utils/staking-miner) using [subxt](https://github.com/paritytech/subxt)
-to avoid hard dependency to each runtime version. It's using a static metadata (metadata.scale) file to generate rust types, instructions how to update metadata can be found [here](#update-metadata)
+The binary itself embeds [static metadata](./artifacts/metadata.scale) to
+generate a rust codegen at compile-time that
```

Comment:
Nit; the line lengths are a bit all over the place (but I know it doesn't make a difference to the output)